packages/server/graphql/public/types/RetrospectiveMeeting.ts

```diff
@@ -14,6 +14,7 @@ const RetrospectiveMeeting: RetrospectiveMeetingResolvers = {
     const res = await dataLoader.get('meetingMembersByMeetingId').load(meetingId)
     return res as RetroMeetingMember[]
   },
+  isLoadingSummary: ({summary}) => summary === '<loading>',
```

Comment:
+1 to ensure we never accidentally render the placeholder on the client, we could add a resolver here
```ts
summary: ({summary}) => summary === '<loading>' ? return null : summary,
```